lua/telescope/actions/init.lua

```diff
@@ -883,6 +883,7 @@ actions.which_key = function(prompt_bufnr, opts)
   opts.border_hl = utils.get_default(opts.border_hl, "TelescopePromptBorder")
   opts.winblend = utils.get_default(opts.winblend, config.values.winblend)
   opts.column_padding = utils.get_default(opts.column_padding, "  ")
+  opts.foldenable = utils.get_default(opts.foldenable, false)
```

Comment:
Don't we just always want it off? Can we just skip creating this new option and just always set it to false on line 1016

Reply:
It is for consistency if people prefer the previous behavior.
I am OK with either way.

Reply:
Neither a folds user nor have I been able to get the which key float to fold :sweat_smile: (though haven't tried again), but it should always not fold. The float is automatically resized to the number of entries and I don't think it could be unfolded.

Sorry, we should have just went with setting it false right away.

Could you please make the change? Then I'll merge :)

Reply:
Updated.